source/agora/node/Validator.d

```diff
@@ -245,13 +245,9 @@ public class Validator : FullNode, API
 
     protected override void startPeriodicDiscovery ()
     {
-        this.taskman.runTask(
-        ()
-        {
-            void discover () { this.network.discover(this.required_peer_keys); }
-            discover();  // avoid delay
-            this.timers ~= this.taskman.setTimer(5.seconds, &discover, Periodic.Yes);
-        });
+        void discover () { this.network.discover(this.required_peer_keys); }
+        discover();  // avoid delay
+        this.timers ~= this.taskman.setTimer(5.seconds, &discover, Periodic.Yes);
```

Comment:
This interval should really be configurable

Reply:
Good point, let me do it now.